abci/cmd/api/validator/validator.go

```diff
@@ -13,11 +13,13 @@ import (
 const (
 	ethAddressRegexString   = `^0x[0-9a-fA-F]{40}$`
 	ethSignatureRegexString = `^0x[0-9a-f]{130}$`
+	hexRegexString          = `^(0x)?([0-9a-fA-F][0-9a-fA-F])+$`
```

Comment:
why need 2 `[0-9a-fA-F]`?

Reply:
Because a single hex digit is half a byte, which is invalid in this case

Reply:
How about using `([0-9a-fA-F]{2})+`?